### PR TITLE
deployment on K8s: cred management and agent env vars

### LINKED ISSUE #86: deployment on k8s: updates of Helm chart

### PR DESCRIPTION
**Pull request by konstan**
- Updated nuvla/job-lite to 3.2.5
- Added after-helm installation note on the approval of the management credential.
- Updated installation documentation.
- Extra env vars for NE agent and k8s cred manager.

NB! The versions of nuvladev/agent and nuvlaedge/kubernetes-credentials-manager are for branches with changes to be validated in the branch for the deployment.

Closes #86

### CODE REVIEW

**Comment by schaubl** [comment]
@konstan I think the values in [values.yaml](https://github.com/nuvlaedge/deployment/pull/85/files#diff-ecd83bf7839981cdbd57e8f45c0572ba32238d451460d1e110db98d05f0179bf) should be updated before I can merge.

**Comment by konstan** [comment]
@schaubl Yes, correct. And [Chart.yaml](https://github.com/nuvlaedge/deployment/pull/85/files#diff-bd7cc4ad832586788bff2714278b2774caaa01bd8a1f5631ba9e9b91e71ebc5d) as well. I will let you know when done.

**Comment by konstan** [comment]
@schaubl Done.

File: helm/nuvlaedge-engine/values.yaml [hunk 1]
```diff
@@ -8,53 +8,64 @@ vpnClientComponentName: vpn-client
...
     repository: nuvla/job-lite
     pullPolicy: Always
-    tag: "3.2.2"
+    tag: "3.2.7"
   kubernetesCredentialsManager:
-    repository: nuvlaedge/kubernetes-credentials-manager
+    repository: nuvladev/kubernetes-credentials-manager
     pullPolicy: Always
-    tag: "1.0.0"
+    tag: "dont-approve"
```
[the remark this answers is not in the document]
Comment (konstan) [reply helm/nuvlaedge-engine/values.yaml]: @schaubl Released and updated here.

File: helm/nuvlaedge-engine/values.yaml
Comment (schaubl) [inline helm/nuvlaedge-engine/values.yaml:1]: @konstan I think the Docker Hub org should be `nuvlaedge` not `nuvladev` because releases are on `nuvlaedge`

Comment (konstan) [reply helm/nuvlaedge-engine/values.yaml:1]: @schaubl Yes, it should be `nuvlaedge`. Fixed.

**Pull request merged**